bears/xml/XMLBear.py

```diff
@@ -1,5 +1,26 @@
+import os
+import requests
+
 from coalib.bearlib.abstractions.Lint import Lint
 from coalib.bears.LocalBear import LocalBear
+from coalib.misc.Shell import escape_path_argument
+from coalib.settings.Setting import path
+
+
+def path_or_url(type1, type2):
```

Comment:
Why does this take in `type1` and `type2` ?
I mean the function name is path or url ... so, it is obviously a path or str right ? Why do you need to askt he user to give it again ?